src/platform/forms-system/test/js/review/ReviewCollapsibleChapter.unit.spec.jsx

```diff
@@ -715,44 +715,6 @@ describe('<ReviewCollapsibleChapter>', () => {
     tree.unmount();
   });
 
-  it('should render a collapsible button with a unique id attribute', () => {
```

Comment:
This is replaced by a `<va-accordion-item>` which I don't believe needs to be tested, as it is library code.